src/httprpc.cpp

```diff
@@ -45,7 +45,7 @@ class HTTPRPCTimer : public RPCTimerBase
 class HTTPRPCTimerInterface : public RPCTimerInterface
 {
 public:
-    HTTPRPCTimerInterface(struct event_base* base) : base(base)
+    HTTPRPCTimerInterface(struct event_base* _base) : base(_base)
```

Comment:
It is not good style to use leading underscore,  such names are reserved (https://stackoverflow.com/questions/228783/what-are-the-rules-about-using-an-underscore-in-a-c-identifier#228797,  for example). We can use underscore at the end of the identifier instead (base_) or leave it as it is now,  may be it is not really important for argument names.

Reply:
I actually believe that both (prefix and postfix underscores) are style ;) But that's unfortunately how it's done in Bitcoin. I'd prefer to stick 1:1 with what comes from the backports to avoid future merge conflicts. When we are at the point where we stop following Bitcoin (maybe after 0.15?), I'm fine with any type of refactorings that diverge from Bitcoin.